qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -697,6 +697,9 @@ def on_current_changed(self, idx):
             log.webview.debug("on_current_changed got called with invalid "
                               "index {}".format(idx))
             return
+        if (self._now_focused is not None and
+                self._now_focused.tab_id is tab.tab_id):
```

Comment:
Isn't this a number comparision (should it be `is` or `==`)? Or, alternatively, maybe it should be `self._now_focused is tab`?

Reply:
Yeah, I guess `self._now_focused is tab` is better.